From f25165a2bcac2ec1b33b541a04ece0eef420674a Mon Sep 17 00:00:00 2001
Message-Id: <f25165a2bcac2ec1b33b541a04ece0eef420674a.1540752056.git.aditya@kobol.io>
In-Reply-To: <3eb15c0c6a0f26e418074cf3be9490a36f9161fd.1540752056.git.aditya@kobol.io>
References: <3eb15c0c6a0f26e418074cf3be9490a36f9161fd.1540752056.git.aditya@kobol.io>
From: Aditya Prayoga <aditya@kobol.io>
Date: Thu, 18 Oct 2018 13:40:13 +0800
Subject: [PATCH 02/11] arm: mvebu: helios4: Add extra environment variable to
 support Armbian

Armbian boot script different env variable name to load armbianEnv.txt
and device tree
---
 include/configs/helios4.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/helios4.h b/include/configs/helios4.h
index c289490..9a9bb4f 100644
--- a/include/configs/helios4.h
+++ b/include/configs/helios4.h
@@ -165,6 +165,8 @@
 	LOAD_ADDRESS_ENV_SETTINGS \
 	"fdtfile=" CONFIG_DEFAULT_DEVICE_TREE ".dtb\0" \
 	"console=ttyS0,115200\0" \
+	"loadaddr=0x02000000\0" \
+	"fdt_addr=" FDT_ADDR_R "\0" \
 	BOOTENV
 
 #endif /* CONFIG_SPL_BUILD */
-- 
2.7.4

